.env

```diff
@@ -0,0 +1,16 @@
+MINIO_VERSION=latest
+MINIO_UI_PORT=9000
+MINIO_REGION=us-east-1
+MINIO_DOMAIN=candig.local
+MINIO_DATA_DIR=/data
+
+MINIO_ACCESS_KEY=miniotest
```

Comment:
@SamarthVP it is bad development practice to add secrets into repos, however in this case we are simply building a test case to prove the concept is viable. Before reaching production, these sorts of secrets are removed...